frontend/packages/dev-console/src/components/import/git/__tests__/GitSection.spec.tsx

```diff
@@ -0,0 +1,143 @@
+import * as React from 'react';
+import { render, waitFor, screen, configure, cleanup, fireEvent } from '@testing-library/react';
+import { Formik, FormikConfig } from 'formik';
+import { act } from 'react-dom/test-utils';
+import { Provider } from 'react-redux';
+import { GitProvider } from '@console/git-service/src';
+import { BuildStrategyType } from '@console/internal/components/build';
+import store from '@console/internal/redux';
+import userEvent from '../../../buildconfig/__tests__/user-event';
+import DevfileStrategySection from '../../devfile/DevfileStrategySection';
+import { GitSectionFormData } from '../GitSection';
+
+configure({ testIdAttribute: 'data-test-id' });
+
+const Wrapper: React.FC<FormikConfig<GitSectionFormData>> = ({ children, ...formikConfig }) => (
+  <Provider store={store}>
+    <Formik {...formikConfig}>
+      {(formikProps) => (
+        <form onSubmit={formikProps.handleSubmit}>
+          {children}
+          <input type="submit" value="Submit" />
+        </form>
+      )}
+    </Formik>
+  </Provider>
+);
+
+afterEach(() => cleanup());
+
+const devStrategyInput = {
```

Comment:
nit: You can make this reusable by putting this in external file like frontend/packages/dev-console/integration-tests/testData/git-import-flow.data.ts

Reply:
Made the changes